source/fundamentals/data-formats/document-data-format-extended-json.txt

```diff
@@ -119,146 +119,280 @@ corresponds to the format of the example you want to see:
-
-   Document doc = Document.parse(ejsonStr);
-   System.out.println(doc);
-
-The output of the code above resembles the following:
-
-.. code-block:: none
-   :copyable: False
-
-   Document{{_id=507f1f77bcf86cd799439011, myNumber=4794261}}
-
-For more information on the driver document classes, see our Fundamentals page
-on :doc:`Documents </fundamentals/data-formats/documents>`.
-
-Using the BSON Library
-~~~~~~~~~~~~~~~~~~~~~~
-
-You can also read an Extended JSON string into Java objects without using
-the MongoDB driver's document classes by using the
-:java-docs:`JsonReader </apidocs/bson/org/bson/json/JsonReader.html>` class.
-This class contains methods to sequentially parse the fields and values
-in any format of the Extended JSON string, and returns them as Java objects.
-MongoDB driver's document classes also use this class to parse Extended JSON.
-
-The following code example shows how you can use ``JsonReader`` to convert
-an Extended JSON string into Java objects:
-
-.. code-block:: java
-
-   String ejsonStr = "{ \"_id\": { \"$oid\": \"507f1f77bcf86cd799439011\"}," +
-                     "\"myNumber\": {\"$numberLong\": \"4794261\" }}}";
-
-   JsonReader jsonReader = new JsonReader(ejsonStr);
-
-   jsonReader.readStartDocument();
-
-   jsonReader.readName("_id");
-   ObjectId id = jsonReader.readObjectId();
-   jsonReader.readName("myNumber");
-   Long myNumber = jsonReader.readInt64();
-
-   jsonReader.readEndDocument();
-
-   System.out.println(id + " is type: " + id.getClass().getName());
-   System.out.println(myNumber + " is type: " + myNumber.getClass().getName());
-
-   jsonReader.close();
-
-The output of this code example resembles the following:
-
-.. code-block:: none
-   :copyable: False
-
-   507f1f77bcf86cd799439011 is type: org.bson.types.ObjectId
-   4794261 is type: java.lang.Long
-
-
-For additional information on the classes and methods mentioned on this
-section, see the following API documentation:
+.. tabs::
 
+   .. tab:: Document Class
+      :tabid: document-class-read
+
+      You can read an Extended JSON string into Java document objects by calling
+      the ``parse()`` static method from either the ``Document`` or ``BsonDocument``
+      class, depending on which object type you need. This method parses the Extended
+      JSON string in any of the formats and returns an instance of that class
+      containing the data.
+      
+      The following example shows how you can use the ``Document`` class to read
+      an example Extended JSON string into a ``Document`` object using the
+      ``parse()`` method:
+      
+      .. code-block:: java
+      
+         String ejsonStr = "{ \"_id\": { \"$oid\": \"507f1f77bcf86cd799439011\"}," +
+                           "\"myNumber\": {\"$numberLong\": \"4794261\" }}}";
+      
+         Document doc = Document.parse(ejsonStr);
+         System.out.println(doc);
+      
+      The output of the code above resembles the following:
```

Comment:
I know this is inherited, but I would lean toward a more casual tone:
```suggestion
      The output of the code above should look something like this:
```

Reply:
Thanks for the suggestion @cbush, I agree that a more casual tone is beneficial here. A quick grep search shows that "resembles the following" is used in approximately 9 different files in the java-docs repos. Do you think it is worth replacing all of the "resembles the following"  instances with more casual language, or just this one instance? I will pull @ccho-mongodb into this comment thread as this is getting outside the scope of this PR.

Reply:
@cbush Is this a nitpick? I don't know if there's an issue with the current text.

Reply:
@biniona-mongodb you can certainly change it if you find a wording that conforms much more strongly to the intent or is something you find reference to in the style guide. I think recommended version is ok -- slightly less formal voice than what we've been using in the Java docs.

Reply:
Changed language throughout java-docs in https://github.com/mongodb/docs-java/pull/57/commits/8fa056f59162a4bb7900e1db00b05c55053390c7 such that "resembles the following" is now different versions of "should look something like this"  when appropriate.

Reply:
@cbush, I provided some feedback on @biniona-mongodb's other [PR]() that seems to conflict with advice here. See [Style Guide](https://docs.mongodb.com/meta/style-guide/writing/use-pronouns-carefully/#this). "this" is too ambiguous without a description. Try "the following output" or "output similar to the following example".  The style guide does note the beginning of a sentence, but it creates the same issues at the beginning or the end of a sentence. From what I understand, it's not a great experience for someone using a screen reader and can create machine translation issues down the line. Using just "the following" creates issues as well.

Reply:
As I mentioned in Slack, my original critique was on the word _resembles_ in phrases like "resembles the following" when "like this" is much simpler and clearer.

That said, I think having the code example directly following a colon should provide enough context.